### PR TITLE
Pull Request for Issue 2453: Change order of pico ammeters.

### PR DESCRIPTION
**Pull request by mButlerDM**
Adjusted PGMBladeCurrentView to have pico ammeters listed in beamline
order.

Fixed an incorrect name of a picoammeter in PGMBeamline.

### CODE REVIEW

**Comment by mButlerDM** [comment]
@XuetaoLiu-CLS Could you review when you a moment? Thanks!

**Comment by dretrex** [comment]
Looks good.
Time Estimate: 1 minute

File: source/beamline/PGM/PGMBeamline.cpp [hunk 1]
```diff
@@ -147,7 +147,7 @@ void PGMBeamline::setupDetectors()
 	exitSlitUpperBladeCurrentBDetector_ = new PGMPicoAmmeter("exitSlitUpperBladeCurrentB", "Exit Slit Upper B", "A1611-4-05:nA:fbk", this);
 
 	entranceSlitLowerBladeCurrentDetector_ = new PGMPicoAmmeter("entranceSlitLowerBladeCurrent", "Entrance Slit Lower", "A1611-3-03:nA:fbk", this);
-	entranceSlitUpperBladeCurrentDetector_ = new PGMPicoAmmeter("entranceSlitUpperBladeCurrent", "Entrance Slit Lower", "A1611-3-04:nA:fbk", this);
+	entranceSlitUpperBladeCurrentDetector_ = new PGMPicoAmmeter("entranceSlitUpperBladeCurrent", "Entrance Slit Upper", "A1611-3-04:nA:fbk", this);
```
Comment (XuetaoLiu-CLS) [inline source/beamline/PGM/PGMBeamline.cpp]: :8ball:

**Comment by XuetaoLiu-CLS** [comment]
Looks good.

Time Estimate: 1 minute

**Pull request merged**